src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,312 @@
+- Galley
+- Mixer
+- Pilot
+- Grafana
+- Prometheus
+- Tracing (Jaeger) 
+- Kiali
+- Mutual TLS
+- Certmanager
+
+With exception of Mutual TLS and Certmanager, all of those are installed. As it is mentioned above mutual TLS can impact ongoing traffic, read the [upstream docs](https://istio.io/docs/tasks/security/mutual-tls/) before enable it. You can configure the components tweaking the values passed to helm. Run `helm inspect install/kubernetes/helm/istio` to display the different options available.
+
+Although Istio can run in almost every Kubernetes cluster regardless of the underlying infrastructure, there are some quirks to consider depending on where the service mesh is deployed.
+
+### AWS and Azure
+
+The actual installation is quite simple thanks to helm. The following command will deploy all Istio components in a new namespace reserved.
+
+```
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system
+```
+
+### On premises (KVM)
+
+The main difference with clouds is the ingress gateway service must be type `NodePort`. Since there is no automatic mechainism to provide an endpoint, the service is exposed in the host underlaying machine. Later, it is explained how to redirect the traffic from the control plane to a second tenant cluster ingress controller.
+
+```
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
+--set gateways.istio-ingressgateway.type=NodePort --set gateways.istio-egressgateway.type=NodePort 
+```
+
+Ensure all components are up and running listing the pods in the istio namespace.
+```
+$ kubectl get pods -n istio-system -w
+```
+
+## Deploy an application
+
+Let's verify Istio is deployed and configure correctly. We can deploy a simple http server that returns `200` code for the `healthz` path of the service.
+
+### Automated sidecar injection
+
+First of all we need to label the namespace to make effective the sidecar injection.
+
+```
+kubectl label namespace default istio-injection=enabled
+```
+
+And now we can install the chart containing the example app.
+
+```
+$ helm registry install quay.io/giantswarm/liveness-chart -n liveness
+```
+
+### Manual sidecar injection
+
+In case you want the manual injection, pull the chart and use Istio command line utility to parse the deployment and insert the proxy.
+
+```
+# Pulling the chart to your local
+$ helm registry pull quay.io/giantswarm/liveness-chart --dest /tmp/
+
+# (CLOUD) Parse the chart template to Kubernetes resources
+$ helm template /tmp/liveness/giantswarm_liveness-chart_XXX/liveness-chart /tmp/liveness.yaml
+
+# (GIANTNETES) Parse the chart template to Kubernetes resources (use the ingress port given by support)
+$ helm template /tmp/liveness/giantswarm_liveness-chart_XXX/liveness-chart /tmp/liveness.yaml --set ingress.port=30010
+
+# Inject the sidecar in the deployment and submit the resources to Kubernetes API
+$ istioctl kube-inject -f /tmp/liveness.yaml | k apply -f -
+```
```

Comment:
I still think what's in the comments here should just be normal article text.

Reply:
forgot those ones